### PR TITLE
[MIG][10] Migration of module stock_picking_show_return

### PR DESCRIPTION
**Pull request by fmdl**

### CODE REVIEW

File: stock_picking_show_return/README.rst [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Usage
+=====
+
+#. Go to any picking that has a return
+#. See the new tab "Returns" appears.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/9.0
```
Comment (rousseldenis) [inline stock_picking_show_return/README.rst]: 10.0

File: stock_picking_show_return/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Show returns on stock pickings',
+    'version': '1.0',
```
Comment (rousseldenis) [inline stock_picking_show_return/__manifest__.py]: Check version. See travis error.

File: stock_picking_show_return/tests/test_stock_picking_show_return.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests import common
```
Comment (rousseldenis) [inline stock_picking_show_return/tests/test_stock_picking_show_return.py]: odoo

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Thanks for migration

File: stock_picking_show_return/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2014-2016 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Show returns on stock pickings',
+    'version': '10.0.1.0',
```
Comment (rousseldenis) [inline stock_picking_show_return/__manifest__.py]: Sorry, I should be clearer : 10.0.1.0.0

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: stock_picking_show_return/README.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+Usage
+=====
+
+#. Go to any picking that has a return
+#. See the new tab "Returns" appears.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/10
```
Comment (rousseldenis) [inline stock_picking_show_return/README.rst]: Oups : '10.0' (test link)

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Typos.

Could you also rename your PR to be more 'attractive' for reviewers ?

**Review by rousseldenis — APPROVED** [review]
:+1: Code review + tested on runbot

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The branch is not correct and contains a lot of commits.

**Comment by pedrobaeza** [comment]
Please include also the recent commit 80869362acda08bcfaa0fd46615a1a8f1376830c with a patch.

**Comment by fmdl** [comment]
I know there lot of commit, but in V10 there are the bad folder : stock_picking_show_return**s** (for V8). Some one make a mistake during the creation of V10 branch (don't use the V9 branch). That why there are all commit. If not good, I can not fix that (I dont know the git command to do what you demand), please remove the stock_picking_show_return**s** folder and add the stock_picking_show_return (V9). After that I can put the new code for V10.

I don't know how add 8086936.

Regards

**Comment by pedrobaeza** [comment]
No, there is no error. When 10.0 branch was created, the module wasn't still renamed. For making correctly, you only need to:

- Make an initial commit renaming the module without s.
- Follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0
- Execute the command `git cherry-pick 80869362acda08bcfaa0fd46615a1a8f1376830c` for picking last commit.

**Comment by fmdl** [comment]
I started from zero : https://github.com/OCA/stock-logistics-workflow/pull/313

**Pull request closed without merging**